src/commands/createNewProject/ProjectCreateStep/PythonProjectCreateStep.ts

```diff
@@ -30,10 +33,34 @@ const defaultRequirements: string = `# DO NOT include azure-functions-worker in
 azure-functions
 `;
 
+const azureWebJobsFeatureFlagsKey = 'AzureWebJobsFeatureFlags';
+
+const defaultFunctionAppSource = `import azure.functions as func
+
+app = func.FunctionApp(auth_level=func.AuthLevel.ANONYMOUS)
+
+# Uncomment to create an HTTP trigger-based function.
+# @app.function_name(name="HttpTrigger1")
+# @app.route(route="hello")
+# def test_function(req: func.HttpRequest) -> func.HttpResponse:
+#    return func.HttpResponse("HttpTrigger1 function processed a request!!!")
+`;
+
 export class PythonProjectCreateStep extends ScriptProjectCreateStep {
     protected gitignore: string = pythonGitignore;
 
+    constructor(private readonly model?: number) {
```

Comment:
JavaScript is getting the same language model treatment. Should we just add the optional parameter to the `ScriptProjectCreateStep`?

Reply:
I'm reluctant to make changes to accommodate Node.js until we see the extent to which there is real overlap.  I suspect that would make sense, but tried my best not to do significant refactoring until we actually get to the point of having more than a single, concrete example of the needed changes.